hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -5829,7 +5829,7 @@ private long replayRecoveredEdits(final Path edits, Map<byte[], Long> maxSeqIdIn
         }
       } catch (EOFException eof) {
         if (!conf.getBoolean(RECOVERED_EDITS_IGNORE_EOF, false)) {
-          Path p = WALSplitUtil.moveAsideBadEditsFile(walFS, edits);
+          Path p = WALSplitUtil.moveAsideBadEditsFile(fs, edits);
           msg = "EnLongAddered EOF. Most likely due to Master failure during "
```

Comment:
I think there is a replacing error...

EnLongAddered should be Encountered, in the past we should have done a replacement from counter to LongAddered...

@alexdongli0829 Minding opening another PR to fix this problem?

Thanks.